src/json_output.rs

```diff
@@ -231,7 +256,24 @@ impl<W: Write> BinXmlOutput<W> for JsonOutput<W> {
                         message: "expected current value to be an object type",
                     })?;
 
-            current_object.insert("#text".to_owned(), value.clone().into());
+            match separate_flag {
```

Comment:
Referring to my comment above, I believe this change is unnecessary.

Reply:
This will still be needed to control if the #text field name should be added or not

Reply:
https://github.com/forensicmatt/evtx/commit/c3b9d8e26693c5da061a4430f000b8c73bf5f943